test/certs/root-ed25519.pubkey.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PUBLIC KEY-----
+MCowBQYDK2VwAyEAGb9ECWmEzf6FQbrBZ9w7lshQhqowtrbLDFw4rXAxZuE=
+-----END PUBLIC KEY-----
```

Comment:
Why was root-ed25519.pubkey.pem added as part of this pull request? Was it by accident?

Reply:
Looks that way...